pom.xml

```diff
@@ -35,12 +35,13 @@
 	</scm>
 
 	<properties>
-		<mwe.version>1.5.0</mwe.version>
-		<mwe2.version>2.11.0</mwe2.version>
-		<tycho.version>1.5.1</tycho.version>
-		<xtext.version>2.19.0</xtext.version>
-		<emf-codegen.version>2.19.0</emf-codegen.version>
-		<ecore-codegen.version>2.19.0</ecore-codegen.version>
+		<mwe.version>1.5.3</mwe.version>
+		<mwe2.version>2.11.3</mwe2.version>
+		<tycho.version>2.0.0</tycho.version>
+		<xtext.version>2.20.0</xtext.version>
```

Comment:
I think Xtext should be 2.22.0 for Eclipse 2020-06, shouldn`t it?

Reply:
Well, there should be no breaking changes, so having 2.22 in your Eclipse with the projects compiled with 2.20 should still be fine. But this only makes sense if there is a reason for not directly upgrading to 2.22.
Even 2.23 was already released, but to ensure compatibility with an Eclipse 2020-06 in standard configuration, I would prefer only to upgrade to at most 2.22.

Reply:
I set it to 2.22.0. I only set it to 2.20.0 because that was the last release Itemis blogged about. But you are right: 2020-06 comes with 2.22.0

Reply:
Great, thank you!